src/parkapi_sources/converters/apcoa/mapper.py

```diff
@@ -0,0 +1,83 @@
+"""
+Copyright 2024 binary butterfly GmbH
+Use of this source code is governed by an MIT-style license that can be found in the LICENSE.txt.
+"""
+
+from datetime import datetime, timezone
+
+from parkapi_sources.models import StaticParkingSiteInput
+from parkapi_sources.models.enums import PurposeType
+
+from .validators import ApcoaNavigationLocationType, ApcoaOpeningHoursWeekdays, ApcoaParkingSiteInput, ApcoaParkingSpaceType
+
+
+class ApcoaMapper:
+    @staticmethod
+    def map_static_parking_site(apcoa_input: ApcoaParkingSiteInput) -> StaticParkingSiteInput:
+        static_parking_site_input = StaticParkingSiteInput(
+            uid=str(apcoa_input.CarParkId),
+            name=apcoa_input.CarparkLongName if apcoa_input.CarparkLongName else apcoa_input.CarparkShortName,
+            lat=next(
+                iter(
+                    location_type_input.GeoCoordinates.Latitude
+                    for location_type_input in apcoa_input.NavigationLocations
+                    if location_type_input.LocationType == ApcoaNavigationLocationType.DEFAULT
+                )
+            ),
+            lon=next(
+                iter(
+                    location_type_input.GeoCoordinates.Longitude
+                    for location_type_input in apcoa_input.NavigationLocations
+                    if location_type_input.LocationType == ApcoaNavigationLocationType.DEFAULT
+                )
+            ),
+            purpose=PurposeType.CAR,
+            type=apcoa_input.CarparkType.Name.to_parking_site_type_input(),
+            has_realtime_data=False,  # TODO: change this as soon as Apcoa API offers realtime data
+            static_data_updated_at=datetime.now(tz=timezone.utc),
```

Comment:
But you do have a last updated at `LastModifiedDateTime`

Reply:
That's right, now added.

Reply:
Why `apcoa_input.LastModifiedDateTime if apcoa_input.LastModifiedDateTime else datetime.now(tz=timezone.utc),`? In which condition `datetime.now(tz=timezone.utc)` will be used?

Reply:
In the case of no entries for ```static_data_updated_at``` to avoid null values.

Reply:
But you validated `LastModifiedDateTime` before so that it will never become `None`:

``` 
    LastModifiedDateTime: datetime = SpacedDateTimeValidator(
        local_timezone=ZoneInfo('Europe/Berlin'),
        target_timezone=timezone.utc,
    )
```

Reply:
That's right.